examples/llm_compression/openvino/tiny_llama_find_hyperparams/requirements.txt

```diff
@@ -5,3 +5,4 @@ openvino==2024.4
 optimum-intel[openvino]>=1.13.0
 transformers>=4.35.2
 onnx<1.16.2
+numpy<2
```

Comment:
why not <2.2?

Reply:
There is used `onnx<1.16.2` that does not support numpy2 and has no upper version limit